components/director/internal/model/scenarios.go

```diff
@@ -38,10 +39,14 @@ var (
 					"type": map[string]interface{}{
 						"const": "string",
 					},
+					"pattern": map[string]interface{}{
+						"const": "^[A-Za-z0-9]([-A-Za-z0-9\\s]*[A-Za-z0-9])$",
```

Comment:
hmm, if `DEFAULT` scenario name is uppercased. Do we want to have any convention, like uppercased snake_case? Maybe we should enforce one convention for naming scenarios, like `camelCase`, `dash-case`, or  uppercased `snake_case`.

Reply:
I would not enforce the convention. I would rather go with the allowed set of characters that can be used.